### DISCUSSION TITLE
Value of 21 not found in GIFNiftyNet.ctbl

### DISCUSSION THREAD

**Discussion by paul-reiners** [TorchIO-project/torchio#498] [Q&A] [answered]
torchio sometimes outputs a region number of 21 which is not found in GIFNiftyNet.ctbl.  Why is this?

**Comment by fepegar** [comment, marked as answer]
Hi, @paul-reiners. You'll need to be more specific. `torchio` is a Python module and doesn't output anything.

I suppose you're using [`highresnet`](https://github.com/fepegar/highresnet). If that's the case, this discussion should happen in that repository. I wouldn't have an answer for that, anyway. Wenqi Li, the person who trained the network using NiftyNet, might be able to help you.

**Reply by paul-reiners** [reply]
Okay, thanks.
